src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Ball randomBall = new Ball();
+        randomBall.setColor();

```

Comment:
Let`s impove your solution. 

First, when you use set method, you should provide argument to this method. 
Second, you can use constructor with variables to create a Ball object like this
new Ball(colorSupplier.getRandomColor(), random.nextInt(MAX_BALL_NUM));

Also. don't forget to [think about which variables should be local in the method and which should be class-level](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#think-about-which-variables-should-be-local-in-the-method-and-which-should-be-class-level)